packages/generator-nitro/generators/app/templates/src/shared/utils/typo/readme.md

```diff
@@ -0,0 +1,44 @@
+# Typo
+
+> Typography util
+
+## Usage
+
+### typography mixin
+
+Writes desired typography definition.
+
+```
+@import 'src/shared/utils/typo/css/typo';
+
+span { @include typography('sans-s') }
+span { @include typography('serif-m') }
+```
+
+Or use a second parameter !== 'all' to only import the definition for specific viewport:
+
+```
+@import 'src/shared/utils/typo/css/typo';
+
+span { @include typography('sans-s', 'md') }
+```
+
+Or use the font-family / font-weight / font-style properties to just get single properties:
+
+```
+@import 'src/shared/utils/typo/css/typo';
+
+span { @include typography('font-family', 'text') }
```

Comment:
maybe we could use a different name here to get a clearar API?

Reply:
you mean mixin name or font-bundle name?

Reply:
extracted to variables